dotnet/ios/InternSampleLogin/AppDelegate.cs

```diff
@@ -0,0 +1,26 @@
+using CodeOnlyDemo;
+
+namespace InternSampleLogin;
+
+[Register ("AppDelegate")]
+public partial class AppDelegate : UIApplicationDelegate
+{
+    // class-level declarations
+    UIWindow? window;
+
+    public override bool FinishedLaunching(UIApplication app, NSDictionary options)
```

Comment:
```suggestion
    public override bool FinishedLaunching (UIApplication app, NSDictionary options)
```
Make sure you put spaces in front of the opening parentheses in your .cs files